pkg/gen/input/ami/funcs.go

```diff
@@ -188,19 +138,3 @@ func scrapeVersionAMI(source io.Reader) (map[string]string, error) {
 
 	return result, nil
 }
-
-var amiTemplate = `// NOTE: This file is generated by devctl. Do not edit.
-package {{ .Package }}
-
-import "encoding/json"
-
-var amiJSON = []byte({{ .AMIInfoString }})
-var amiInfo = map[string]map[string]string{}
-
-func init() {
-	err := json.Unmarshal(amiJSON, &amiInfo)
-	if err != nil {
-		panic(err)
-	}
-}
-`
```

Comment:
This is now in `file` package to decouple templating from computing template data.